qa/workunits/cephtool/test.sh

```diff
@@ -1820,10 +1820,13 @@ function test_mon_pg()
   ceph pg scrub 1.0
 
   ceph osd set-full-ratio .962
+  ceph osd get full-ratio
```

Comment:
This is not validating the output, nor whether the command even succeeds. The same goes for all other `get` commands being tested.